modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,127 @@ The properties for broker-wide throughput quota balancing are configured at the
 
 == Client throughput limits
 
-Similar to <<broker-wide-throughput-limits>> but for clients, Redpanda provides configurable throughput quotas that apply to an individual client or a group of clients.
+Redpanda provides configurable throughput quotas that apply to an individual client or a group of clients. You can apply a quota for an individual client based on an exact match with its `client_id`, or a group of clients based on IDs that start with a given prefix. 
+
+As of v24.2, client throughput quotas are compatible with the https://cwiki.apache.org/confluence/display/KAFKA/KIP-546%3A+Add+Client+Quota+APIs+to+the+Admin+Client[Kafka API^] (AlterClientQuotas and DescribeClientQuotas), and are separate from quotas configured through cluster configuration in earlier Redpanda versions. The client throughput quotas no longer apply on a per-shard basis, and now limit the rates across a Redpanda broker's node. The quotas are neither shared nor balanced between brokers.
+
+Redpanda supports the following Kafka API-based quota types on clients:
+
+|===
```

Comment:
@pgellert Do the Kafka quota types correspond exactly with existing cluster config quota types and do we need to map those out here as well? @mattschumpert

Reply:
Yes, I've added the mapping in this comment: https://github.com/redpanda-data/docs/pull/592#discussion_r1670312568. Feel free to move it wherever you think it makes the most sense.